# NOT USED NOT USED NOT USED, TO CONFIRM
# DEPRECATED TO FOLLOWBEHAVIOUR SINCE IT REFACTORS EVERYTHING RELATED TO FOLLOW MODES

#AVAILABLE AT /bea/follow_mode

#VARIABLES
int8 follow_mode

# CONTANTS
int8 NO_FOLLOW = 0
int8 FOLLOW_FACES = 1
int8 FOLLOW_LEFT_GRIPPER = 2
int8 FOLLOW_RIGHT_GRIPPER = 3
int8 FIND_OBJECTS = 4
